MarathonRecomp/patches/fps_patches.cpp

```diff
@@ -12,3 +12,9 @@ PPC_FUNC(sub_82587AA8)
 
     __imp__sub_82587AA8(ctx, base);
 }
+
+void HFR_POSTURE_STEER_FIX_01(PPCRegister& c_rotation_speed, PPCRegister& stack)
```

Comment:
Rename `HFR_POSTURE_STEER_FIX_01` to `PostureControlRotationSpeedFix` and `c_rotation_speed` to `rotationSpeed`.